packages/jest-mock-apollo/src/test/MockApolloLink.test.ts

```diff
@@ -27,14 +27,19 @@ describe('MockApolloLink', () => {
   it('returns error message with empty mock object', async () => {
     const mockApolloLink = new MockApolloLink({}, schema);
 
-    const result = await new Promise(resolve => {
-      mockApolloLink.request(mockRequest).subscribe(resolve);
-    });
-
-    expect(result).toMatchObject({
-      message:
-        "Can’t perform GraphQL operation 'Pets' because no valid mocks were found (it looks like you tried to provide data directly to the mock GraphQL client. You need to provide your fixture on the key that matches its operation name. To fix this, simply change your code to read 'mockGraphQLClient({Pets: yourFixture}).'",
-    });
+    await expect(
+      new Promise((_, reject) => {
+        mockApolloLink.request(mockRequest).subscribe({
+          next() {},
+          error(err) {
+            reject(err);
+          },
+          complete() {},
+        });
+      }),
+    ).rejects.toThrow(
+      "Can’t perform GraphQL operation 'Pets' because no valid mocks were found (it looks like you tried to provide data directly to the mock GraphQL client. You need to provide your fixture on the key that matches its operation name. To fix this, simply change your code to read 'mockGraphQLClient({Pets: yourFixture}).'",
```

Comment:
In your travels, did you notice if it would be possible anywhere to have these kinds of things actually fail the test? I wonder if we can maybe propagate it to the method where we `resolveAllGraphQL` so that that part rejects?

Reply:
I'm certain there's a way to fail fast, but I haven't found it yet :/

Unfortunately, many of these errors come from tests that never attempt to `resolveAllGraphQL`, so that wouldn't provide comprehensive coverage.